index.js

```diff
@@ -59,68 +49,51 @@ const handleOptionsElementOnClick = (tabName) => {
 const removeSubMenuElement = () => document.getElementById(ELEMENT_IDS.subMenu)?.remove()
 //
 
-const renderSubMenu = (options, param2, index) => {
+// GETTERS
+const getMainElement = () => document.querySelector("main")
+const getScreenWrapperElement = () => document.getElementById(ELEMENT_IDS.screenWrapper)
+//
+
+const renderSubMenu = (options, buttonName, index) => {
     removeSubMenuElement()
-    handleOptionsElementOnClick(param2)
+    handleOptionsElementOnClick(buttonName)
 
     const mainElement = getMainElement()
 
     const subMenuElement = createElement("div", {
         id: ELEMENT_IDS.subMenu,
-        className: "options-wrapper2"
+        className: "options-wrapper"
     }, mainElement)
 
-    const elementHeight = "40.67px"
-    const gapHeight = "20px"
-
-    subMenuElement.style.top = `calc(${index + 1} * ${elementHeight} + ${index + 1} * ${gapHeight})`
+    subMenuElement.style.top = `calc(${index} * ${ELEMENT_HEIGHT} + ${index} * ${GAP_HEIGHT})`
 
-    options.forEach(({ id, innerText, onclick }) => createElement("button", {
-        id,
+    options.forEach(({ innerText, onclick }) => createElement("button", {
         className: "sub-options",
         innerText,
         onclick
     }, subMenuElement))
 }
 
-
-// TODO: make 1 function from the 2
-const renderDigitalProgrammeList = () => {
-    removeSubMenuElement()
-    handleOptionsElementOnClick("digitalProgrammeList")
-    console.log("Digital Programme List")
-}
-
-const renderDigitalEpg = () => {
-    removeSubMenuElement()
-    handleOptionsElementOnClick("digitalEpg")
-    console.log("Digital EPG")
-}
-
 // DATA
 const HOME_MENU_BUTTONS_ARRAY = [
     {
         id: ELEMENT_IDS.internetContentsTab,
-        param2: "internetContents",
+        buttonName: "internetContents",
```

Comment:
Instead of introducing new key `innerText` could be transformed

Reply:
Don't really know how you mean. I need buttonName to handle disable buttons.